.github/workflows/pkgdown.yaml

```diff
@@ -42,32 +42,12 @@ jobs:
           needs: website
 
       - name: Build site
-        # - target_ref gets the ref from a different variable, depending on the event
-        # - override allows us to set the pkgdown mode and version_label
-        #   - mode: release is the standard build mode, devel places the site in /dev
-        #   - version_label: 'light' and 'success' are CSS labels for Bootswatch: Cosmo
-        #                    https://bootswatch.com/cosmo/
-        # - we use pkgdown:::build_github_pages to build the site because of an issue in pkgdown
-        #   https://github.com/r-lib/pkgdown/issues/2257
-        run: |
-          target_ref <- "${{ github.event_name == 'pull_request' && github.base_ref || github.ref }}"
-          override <- if (target_ref == "main" || target_ref == "refs/heads/main") {
-            list(development = list(mode = "release", version_label = "light"))
-          } else if (target_ref == "dev" || target_ref == "refs/heads/dev") {
-            list(development = list(mode = "devel", version_label = "success"))
-          } else {
-            stop("Unexpected target_ref: ", target_ref)
-          }
-          pkg <- pkgdown::as_pkgdown(".", override = override)
-          cli::cli_rule("Cleaning files from old site...")
-          pkgdown::clean_site(pkg)
-          pkgdown::build_site(pkg, preview = FALSE, install = FALSE, new_process = FALSE)
-          pkgdown:::build_github_pages(pkg)
+        run: pkgdown::build_site_github_pages(new_process = FALSE, install = FALSE)
```

Comment:
So all the previous code handled made custom dev docs for the `dev` branch, rather than basing it on `x.y.z.9000` version numbers. I honestly would prefer to move to the latter, default model, but we'll need to adjust our workflows a bit.

Reply:
I'll bring this back. I don't think there's a way to change with the `dev` branch unfortunately. But hopefully we don't need the color handling any more.